src/versionHelper.js

```diff
@@ -0,0 +1,27 @@
+import * as PIXI from 'pixi.js';
+
+const pixiVersionHelper = {
+  version: PIXI.VERSION,
+  semanticVersion() {
+    const semVer = this.version.match(/([0-9]+)\.([0-9]+)\.([0-9]+)/);
+    if (semVer.length < 3) {
+      return [0, 0, 0];
+    }
+    const major = parseInt(semVer[1], 10);
+    const minor = parseInt(semVer[2], 10);
+    const patch = parseInt(semVer[3], 10);
+    return [major, minor, patch];
```

Comment:
I suggests that returned value's type is object instead of array.
```
return { major, minor, patch }
```
What do you think this suggestion？

Reply:
Fixed it !